jdaviz/helpers.py

```diff
@@ -0,0 +1,28 @@
+import pathlib
+from specutils import Spectrum1D
+
+from .app import Application
+
+class ConfigHelper():
+    """The Base Helper Class
+    Provides shared abstracted helper methods to the user
+    """
+    def __init__(self, configuration=None):
+        self._app = Application(configuration=configuration)
+
+
+class SpecViz(ConfigHelper):
+    def __init__(self):
+        super().__init__(configuration="specviz")
+
+    def load_data(self, data, data_label, format=None):
+        try:
+            if pathlib.Path(data).is_file():
```

Comment:
This is missing an `else`, which is a bit problematic because it will the happily run if the user gives it something but just not load the data.  I suggest a few if/elif clauses to cover the following cases:

1. if it's a `Spectrum1D`, load it as the data
2. if it's a file, do exactly what you're doing here
3. if it's a `SpectrumCollection` raise an error that says "you probably want a Spectrum1D" or something
4. if it's anything else, raise an exception that it's an unrecognized data type.

Reply:
@eteq just wrapped this one up, and tests are passing. See if this last commit works to your liking? Thanks!